apps/web/src/components/PaginatedTable/components/RollupFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from "react";
+
+import { Dropdown } from "~/components/Dropdown/Dropdown";
+import type { DropdownProps } from "~/components/Dropdown/Dropdown";
+import { RollupIcon } from "~/components/RollupIcon";
+import type { Rollup } from "~/types";
+
+const rollups: Rollup[] = [
+  "base",
+  "mode",
+  "scroll",
+  "arbitrum",
+  "blast",
+  "boba",
+  "camp",
+  "kroma",
+  "linea",
+  "metal",
+  "optimism",
+  "optopia",
+  "paradex",
+  "pgn",
+  "starknet",
+  "taiko",
+  "zksync",
+  "zora",
+];
+
+const ROLLUP_OPTIONS: DropdownProps["options"] = rollups.map((rollup) => {
+  const icon = <RollupIcon rollup={rollup} />;
+  return {
+    value: rollup,
+    label: rollup.charAt(0).toUpperCase() + rollup.slice(1),
+    prefix: icon === null ? <div className="h-4 w-4"></div> : icon,
+  };
+});
```

Comment:
I think we should keep the `Option` implementation for `RollupFilter` decoupled from the `RollupBadge` because they are different components:

The badge is used here 
![image](https://github.com/user-attachments/assets/04773a30-393b-4560-9c33-f3a65b85388d)
where a different background/text color is applied for all the rollups. This will cause a dropdown component full of different options styles.

Moreover, this implementation follows the [ new refactor](https://github.com/Blobscan/blobscan/pull/491) for the `Dropdown` component where we want to maintain consistency between all the dropdown currencies in the application. If we pass a whole component like `RollupBadge`, we'll have non-homogeneous implementations

Reply:
Let's keep it like it is now